### PR TITLE
INT-1124: add new connector BZpay

### PR DESCRIPTION
**Pull request by SergeKoTnk**

### CODE REVIEW

File: data/vendors.json [hunk 1]
```diff
@@ -28396,6 +28396,18 @@
       "uk": "GlobalTransactionCharge"
     }
   },
+  {
+    "code": "bzpay",
+    "status": "active",
+    "links": {
+      "website": "https://www.bzpay.com.mx/"
```
Comment (TonyTramb) [inline data/vendors.json:28403]: image
logo + icon

File: data/vendors.json [hunk 2]
```diff
@@ -28396,6 +28396,18 @@
...
+    },
+    "name": {
+      "en": "bzpay",
```
Comment (TonyTramb) [inline data/vendors.json:28406]: BZPAY

File: data/payment_providers.json [hunk 3]
```diff
@@ -17545,6 +17545,19 @@
       "payment_card"
     ]
   },
+  {
+    "status": "active",
+    "code": "bzpay",
+    "vendor": "bzpay",
+    "name": {
+      "en": "bzpay",
```
Comment (TonyTramb) [inline data/payment_providers.json:17553]: BZPAY

File: data/vendors.json (hunk 1 above)
Comment (SergeKoTnk) [reply data/vendors.json:28403]: added

File: data/vendors.json (hunk 2 above)
Comment (SergeKoTnk) [reply data/vendors.json]: renamed

File: data/payment_providers.json (hunk 3 above)
Comment (SergeKoTnk) [reply data/payment_providers.json]: renamed

**Review by TonyTramb — APPROVED** [review]

**Pull request merged**